Assets/Tests/PlayMode/UI/HorizontalBarTest.cs

```diff
@@ -47,54 +46,54 @@ public void HorizonalBarTestSimplePasses()
 
             int damageGiven = 25;
 
-            float widthBarBefore = (float)healthBar.GetComponent<RectTransform>().sizeDelta.x;
+            float widthBarBefore = (float) healthBar.GetComponent<RectTransform>().sizeDelta.x;
 
             HorizontalBar horizontalBar = updateBar.GetComponent<HorizontalBar>();
 
             //TODO: peut-être faire un for
 
-            float expectedBarAfter = (float)updateBar.GetComponent<RectTransform>().rect.width * ((float)(myHero.CurrentHealth - damageGiven) / (float)myHero.GetStats().MaxHealth);
+            // On répète le processus quatres fois avec 25 dégats de données
```

Comment:
in english

Reply:
@wilfried7 trop de commentaires en français à update